cornflow/config.py

```diff
@@ -54,15 +54,24 @@ class Testing(object):
     AUTH_TYPE = int(os.getenv("AUTH_TYPE", AUTH_DB))
 
     # LDAP configuration
-    LDAP_PROTOCOL_VERSION = 3
-    LDAP_BIND_PASSWORD = "adminldap"
-    LDAP_BIND_DN = "cn=admin,dc=example,dc=org"
-    LDAP_USE_TLS = False
-    LDAP_HOST = "ldap://35.205.18.159:389"
-    LDAP_USERNAME_ATTRIBUTE = "cn"
-    LDAP_USER_BASE = "ou=user,dc=example,dc=org"
-    LDAP_EMAIL_ATTRIBUTE = "mail"
-    LDAP_USER_OBJECT_CLASS = "inetOrgPerson"
+    LDAP_HOST = os.getenv("LDAP_HOST")
+    LDAP_BIND_DN = os.getenv("LDAP_BIND_DN")
+    LDAP_BIND_PASSWORD = os.getenv("LDAP_BIND_PASSWORD")
+    LDAP_USERNAME_ATTRIBUTE = os.getenv("LDAP_USERNAME_ATTRIBUTE")
+    LDAP_USER_BASE = os.getenv("LDAP_USER_BASE")
+    LDAP_EMAIL_ATTRIBUTE = os.getenv("LDAP_EMAIL_ATTRIBUTE")
+    LDAP_USER_OBJECT_CLASS = os.getenv("LDAP_USER_OBJECT_CLASS")
+
+    LDAP_GROUP_OBJECT_CLASS = os.getenv("LDAP_GROUP_OBJECT_CLASS")
+    LDAP_GROUP_ATTRIBUTE = os.getenv("LDAP_GROUP_ATTRIBUTE")
+    LDAP_GROUP_BASE = os.getenv("LDAP_GROUP_BASE")
+    LDAP_GROUP_TO_ROLE_SERVICE = os.getenv("LDAP_GROUP_TO_ROLE_SERVICE")
+    LDAP_GROUP_TO_ROLE_ADMIN = os.getenv("LDAP_GROUP_TO_ROLE_ADMIN")
+    LDAP_GROUP_TO_ROLE_VIEWER = os.getenv("LDAP_GROUP_TO_ROLE_VIEWER")
+    LDAP_GROUP_TO_ROLE_PLANNER = os.getenv("LDAP_GROUP_TO_ROLE_PLANNER")
+
+    LDAP_PROTOCOL_VERSION = int(os.getenv("LDAP_PROTOCOL_VERSION", 3))
+    LDAP_USE_TLS = os.getenv("LDAP_USE_TLS")
```

Comment:
if you're using environment variables, then just make `Testing` inherit from `MainEnvVars` as the other two. You do not need these lines then

Reply:
Will change it